### ISSUE TITLE
About the performance of training

### ISSUE THREAD

**Issue by swt199211** [lorenmt/reco#8]
I'm sorry to bother you. I see that your code works very well, so I want to try to run it. However, when I perform reco and perform semi supervision on 600 tags, the result is only 0.6719. Due to the problem of GPU memory, I changed the batchsize from 10 to 6, and I haven't changed other parameters. Why is the result gap so large? I can't figure out the reason now. I'd like to ask you for advice. Thank you very much

**Comment by swt199211** [comment]
Thank you again for your reply. I have run out of the same results as in your paper. I am now trying to change your semi supervised method to a binary infrared small target semantic segmentation network. Only the detection of target and background, the infrared target is very small, and some are just a few pixels. Do you think your network is also applicable? Or what should I pay attention to? I have no experience. I have just done this direction in the first grade of graduate student, and no one in the group has done this direction. If you can reply to me, I will be very grateful.

**Comment by lorenmt** [comment]
I think it should work, as long as you have some accurate training images.

Since DeepLabv3+ would down-sample images by 4 times, it may sometimes completely ignore some small objects.

So you need to make sure that even after down-sample 4 times, the small objects are still detectable. Otherwise, you need to pre-process your images to make them easier to train.

**Comment by swt199211** [comment]
Thank you again for your reply. The network I use is dedicated to small infrared targets. I have now changed your semi supervision to this network, but I have trained that the recoloss has always maintained the initial size and has not decreased. What is the reason for this? My current level is not very good, and I haven't found the problem. I see that your original training recoloss will be reduced by about 2

**Comment by lorenmt** [comment]
I don't understand this question. Could you provide more details?>

**Comment by swt199211** [comment]
Hello, let me rephrase my question. At the end of the network, you lead to a branch of feature representation, namely semantic segmentation and feature representation. The way you use comparative learning to find features is judged by the results of semantic segmentation branches, and then you take the feature vector at the corresponding position in the feature representation branch. But is this correspondence reasonable? Why can the network learn this correspondence? If the network does not learn this correspondence, it will affect the final result?
